apps/mobile/src/features/token/types.ts

```diff
@@ -8,3 +14,61 @@ export interface TokenDetailsProps {
 export interface OnPressTokenDetails {
   onPress?: (tokenDetails: TokenDetailsProps) => void;
 }
+
+export type SupportedAssetProtocol = 'nativeBtc' | 'nativeStx' | 'sip10';
+
+export type TokenBalance =
+  | AccountQuotedBtcBalance
+  | AddressQuotedStxBalance
+  | Sip10Balance
+  | RuneBalance;
+
+export function isAccountQuotedBtcBalance(
```

Comment:
I added these type guards for balance type as it makes it cleaner to handle multiple types of protocol balances more easily via `getBalance`